dataland-frontend/src/components/forms/parts/elements/derived/ActivitySelector.vue

```diff
@@ -43,13 +43,14 @@
       ref="multiSelectFormFieldRef"
       dataTest="selectNaceCodes"
       name="naceCodes"
-      :emptyMessage="`No NACE code available for ${selectedActivities.name} Activity`"
-      :validation="naceCodesForActivities.length ? 'required' : ''"
+      :emptyMessage="`No NACE code available for ${selectedActivities?.name} Activity`"
```

Comment:
Can selectedActivities be null here?

Reply:
Klar, insbesondere, wenn du die Liste der Activities neu aufmachst, ist ja nix vorausgewählt.